src/main/java/de/komoot/photon/utils/ConvertToJson.java

```diff
@@ -77,6 +77,9 @@ private String getLocalised(Map<String, Object> source, String fieldName, String
             return map.get(lang);
         }
 
+        if (!map.containsKey("default")) {
+            return map.get(map.keySet().toArray()[0]); // Returning first value in the map
```

Comment:
We really should define an order here. Right now you might get a random localized name, if I understand the code right. I suggest: housename, int, loc, reg, alt, old

Also, note that again only NAME has other names than the default and localised ones. So a special handling of the name would be a good idea.

Reply:
The latest commit handles the precedence order you suggested. This handling is only for the name field but in the same `getLocalised()` function. (If a separate function is used, most of the code will be repeated, so I did not go for it)